### DISCUSSION TITLE
Hi, how many emails sent by this software will go into the spam of big companies such as Gmail, outlook, etc.?

### DISCUSSION THREAD

**Discussion by zhushen12580** [postalserver/postal#2998] [Help with using Postal]
refer the title or subject

**Comment by willpower232** [comment]
I'm afraid there isn't a straightforward answer. If you send emails from an IP address they deem not reputable, or your email is actually a spam email, or they just decide not to accept your email, then your recipients won't receive your messages. Postal does its best but sometimes there is nothing you can do.
